kod/object/active/holder/nomoveon/battler/monster/towns/Goad.kod

```diff
@@ -1243,16 +1243,22 @@ messages:
    SomethingEntered(what=$)
    {
 
-      if IsClass(what,&admin) and not send(what,@checkplayerflag,#flag=PFLAG_MORPHED)
-         and not send(what,@checkplayerflag,#flag=PFLAG_ANONYMOUS)
+      if IsClass(what,&Admin)
```

Comment:
This is kind of ridiculous to have the same set of checks in 4 different places.  Turn it into a function so that it's just in one place.